Libs/Path/StuckDetector.cs

```diff
@@ -84,35 +84,36 @@ public async Task Unstick()
             {
                 int strafeDuration = (int)(1000 + (((double)actionDurationSeconds * 1000) / 12));
 
-                if (strafeDuration > 5)
+                if (strafeDuration > 20000)
                 {
-                    strafeDuration = 5;
+                    strafeDuration = 20000;
                 }
 
-                if (actionDurationSeconds > 60)
+                if (actionDurationSeconds > 20)
                 {
-                    // back up a bit
-                    wowProcess.SetKeyState(ConsoleKey.DownArrow, true, false, "StuckDetector");
+                    // back up a bit, added "remove" move forward
+                    wowProcess.SetKeyState(ConsoleKey.DownArrow, true, false, "StuckDetector_back_up");
+					wowProcess.SetKeyState(ConsoleKey.UpArrow, false, false, "StuckDetector");
                     await Task.Delay(strafeDuration);
                     wowProcess.SetKeyState(ConsoleKey.DownArrow, false, false, "StuckDetector");
                 }
                 this.stopMoving?.Stop();
 
-                // stuck for 30 seconds
+                // stuck for 20 seconds
                 var r = random.Next(0, 100);
                 if (r < 50)
                 {
                     logger.LogInformation($"Trying to unstick by strafing left for {strafeDuration}ms");
-                    wowProcess.SetKeyState(ConsoleKey.Q, true, false, "StuckDetector");
+                    wowProcess.SetKeyState(ConsoleKey.A, true, false, "StuckDetector");
```

Comment:
This isn't strafing anymore it is turning.

Reply:
Ah this change is not what I wanted to highlight hehe, it's just because I have strafe on those. Ignore this